scalafmt-tests/src/test/scala/org/scalafmt/config/ConfigDialectOverrideTest.scala

```diff
@@ -0,0 +1,123 @@
+package org.scalafmt.config
+
+import munit.FunSuite
+
+class ConfigDialectOverrideTest extends FunSuite {
```

Comment:
this file tests your dialect map indirectly, mostly focusing on testing the overrides.

I'd add tests which obtain the map and directly check existence of some keys and apply their value function to some dialect and some argument.

Reply:
is this the only comment still to be addressed?

Reply:
Just pushed out the commit with the new test - I must have somehow overlooked this before